docset/winserver2022-ps/grouppolicy/Get-GPO.md

```diff
@@ -124,19 +143,24 @@ Accept wildcard characters: False
 ```
 
 ### -Domain
+
 Specifies the domain for this cmdlet.
 You must specify the fully qualified domain name (FQDN) of the domain.
 
 For the **Get-GPO** cmdlet, the GPO (or GPOs) to that this cmdlet gets must exist in this domain.
 
-If you do not specify the *Domain* parameter, the domain of the user that is running the current session is used.
-If the cmdlet is being run from a computer startup or shutdown script, the domain of the computer is used.
+If you do not specify the **Domain** parameter, the domain of the user that is running the current
+session is used.
+If the cmdlet is being run from a computer startup or shutdown script, the domain of the computer
+is used.
 For more information, see the Notes section in the full Help.
 
-If you specify a domain that is different from the domain of the user that is running the current session (or, for a startup or shutdown script, the computer), a trust must exist between that domain and the domain of the user or the computer.
+If you specify a domain that is different from the domain of the user that is running the current
+session (or, for a startup or shutdown script, the computer), a trust must exist between that
+domain and the domain of the user or the computer.
 
-You can also refer to the *Domain* parameter by its built-in alias, domainname.
-For more information, see about_Aliases.
+You can also refer to the **Domain** parameter by its built-in alias, **domainname.**
+For more information, see **about_Aliases.**
```

Comment:
Here we can actually link directly to the documentation:

```suggestion
For more information, see [about_Aliases](/powershell/module/microsoft.powershell.core/about/about_aliases).
```

When we link to articles on the platform, we use site-relative links. The quickest way to find these is to navigate to the article and trim the URL. In this case, it goes from:

```
https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_aliases?view=powershell-7.3
```

to

```
/powershell/module/microsoft.powershell.core/about/about_aliases
```

By trimming the site root and the language-identifier (so folks reading in another language can be automatically redirected) from the beginning of the URL. We also trim the `view` query parameter from the end.